packages/kbn-es-archiver/README.mdx

```diff
@@ -0,0 +1,17 @@
+---
+id: kibDevDocsOpsEsArchiver
+slug: /kibana-dev-docs/ops/es-archiver
+title: "ES Archiver"
+description: A tool which helps developers capture and restore ES indexes
+tags: ['kibana', 'dev', 'contributor', 'operations', 'ci']
+---
+
+The ES Archiver is a service primarily used by the Functional Tests to load up ES indexes using the bulk API which makes the archives more resilient to ES upgrades and easier to inspect locally because they are just plain text JSON files (though they're often compressed).
```

Comment:
But these files are not actually json files no?

Reply:
Well, they're files that include JSON so they're modifiable in a text editor in a way that ES snapshot/restore files are not.

Reply:
Updates in [`2f7cf97` (#139866)](https://github.com/elastic/kibana/pull/139866/commits/2f7cf97b59649d60260c62a2048c8eb097611bfd)